build.env

```diff
@@ -47,3 +47,7 @@ ROOK_VERSION=v1.3.9
 #CEPH_CSI_RUN_ALL_TESTS=true
 E2E_TIMEOUT=60m
 DEPLOY_TIMEOUT=10
+
+# This var will be used by CentOS CI to build the image
```

Comment:
It is not only CentOS CI, but standard `make`. Maybe move it more to the top of the file?